src/cli.cr

```diff
@@ -44,7 +44,6 @@ module Shards
         self.skip_postinstall = true
       end
       opts.on("--local", "Don't update remote repositories, use the local cache only.") { self.local = true }
-      opts.on("--ignore-crystal-version", "Do not enforce crystal version restrictions on shards.") { self.ignore_crystal_version = true }
```

Comment:
This flag must not become invalid, else we're just adding another breakage on top.

https://github.com/search?q=%22ignore+crystal+version%22&type=code